src/Collection/CollectionBuilder.php

```diff
@@ -71,6 +71,21 @@ public function __construct($commandFile)
     }
 
     /**
+     * @param int $verbosityLevel
+     *
+     * @return $this
+     */
+    public function setVerbosityLevel($verbosityLevel)
+    {
+        $this->verbosityLevel = $verbosityLevel;
+        if (method_exists($this->currentTask, 'setVerbosityLevel')) {
```

Comment:
I don't love checking for the method. The alternative is to add `setVerbosityLevel()` and `getVerbosityLevel()` to `TaskInterface`